persistent/Database/Persist/Quasi.hs

```diff
@@ -415,10 +413,14 @@ Unfortunately, we can't use this to create Haddocks for you, because <https://gi
 -}
 module Database.Persist.Quasi
     ( parse
-    , PersistSettings (..)
+    , PersistSettings
     , upperCaseSettings
     , lowerCaseSettings
+    , setPsUseSnakeCaseForiegnKeys
```

Comment:
We should export the `FKName` type and constructor here too, for use with the setter.